SimG4Core/Application/python/g4SimHits_cfi.py

```diff
@@ -575,6 +575,15 @@
 run2_common.toModify( g4SimHits.HFShowerLibrary, FileName = 'SimG4CMS/Calo/data/HFShowerLibrary_npmt_noatt_eta4_16en_v4.root' )
 run2_common.toModify( g4SimHits.HFShower, ProbMax = 0.5)
 
+
+from Configuration.Eras.Modifier_ctpps_2016_cff import ctpps_2016
+ctpps_2016.toModify( g4SimHits.Generator, HepMCProductLabel = 'LHCTransport')
+ctpps_2016.toModify( g4SimHits.Generator, MinEtaCut = cms.double(-13.0))
+ctpps_2016.toModify( g4SimHits.Generator, MaxEtaCut = cms.double( 13.0))
+ctpps_2016.toModify( g4SimHits.SteppingAction, MaxTrackTime = cms.double(2000.0))
+ctpps_2016.toModify( g4SimHits.StackingAction, MaxTrackTime = cms.double(2000.0))
```

Comment:
This could also be done by
```suggestion
ctpps_2016.toModify(g4SimHits,
    Generator = dict(
        HepMCProductLabel = 'LHCTransport',
        MinEtaCut = -13.0,
        MaxEtaCut = 13.0
    ),
    SteppingAction = dict(
        MaxTrackTime = 2000.0,
        MaxTrackTime = 2000.0
    )
)
```

Reply:
The build error is due to an error in the suggested change in https://github.com/cms-sw/cmssw/pull/29916#discussion_r428796936.

I accept it without noticing the problem. As in the original code, the MaxTrackTime need to be changed in SteppingAction AND StackingAction. Can I change it using the web interface?

Reply:
Sorry, apparently I didn't look carefully enough.

Reply:
No problem, the words are pretty similar, I have done the same mistake the first time... I changed it by means of the web interface and submitted. Is this way ok with you? It is much straightforward.

Reply:
Yes, it looks fine now, thanks.